<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-7ARKDEF" failures="1" tests="6" name="Test" time="0.138" errors="0" timestamp="2020-11-29T18:26:47 EST">
  <testcase classname="test.day1" name="firstTest" time="0.013"/>
  <testcase classname="test.day1" name="secondTest" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.day1.secondTest(day1.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- secondTest -->
  <testcase classname="test.day2" name="fourthTest" time="0.002"/>
  <testcase classname="test.day2" name="thirdTest" time="0.003"/>
  <testcase classname="test.day3" name="fifthTest" time="0.001"/>
  <testcase classname="test.day3" name="sixthTest" time="0.002"/>
</testsuite> <!-- Test -->
